README.md

```diff
@@ -55,21 +56,26 @@ There are many ways to contribute to the project:
 
 ### Usage Reporting ###
 
-BentoML by default collects anonymous usage data using [Amplitude](https://amplitude.com/). 
-It only collects BentoML library's own actions and parameters, no user or model data will be collected. 
-Here is the [code](https://github.com/bentoml/BentoML/blob/main/bentoml/_internal/utils/usage_stats.py) that does it.
+By default, BentoML anonymously collect usage data, and here's the [code](./bentoml/_internal/utils/analytics/usage_stats.py) for it. We rely heavily of
```

Comment:
BentoML collects anonymous usage data that helps our team to improve the product. Only BentoML's internal API calls and CLI commands are being reported and we striped out any potentially sensitive information as much as possible, we will never collect user code, model data, model name or stacktrace. Here's the [code] for usage tracking.

You can opt-out of usage tracking by the `--do-not-track` CLI option:
```bash
> bentoml [command] --do-not-track
```

Or by setting environment variable `BENTOML_DO_NOT_TRACK=True`:
```bash
> export BENTOML_DO_NOT_TRACK=True
```

Reply:
done